### DISCUSSION TITLE
Font source

### DISCUSSION THREAD

**Discussion by rass29** [mearashadowfax/ScrewFast#113] [Q&A] [answered]
Hi, I'm using this template, but I'm not sure how to modify the font family, can you help me with that?

**Comment by mearashadowfax** [comment, marked as answer]
> Hi, I'm using this template, but I'm not sure how to modify the font family, can you help me with that?

Hi @rass29! By default, Tailwind uses a cross-browser font stack. If you want to change the font family, you can do so by editing the `tailwind.config.js` file and your CSS file. For more details, you can check the docs here: [https://tailwindcss.com/docs/font-family#using-custom-values](https://tailwindcss.com/docs/font-family#using-custom-values)
